operations/operations/publishing/publishing-guidelines/brand-and-visual-design-guidelines.md

```diff
@@ -113,7 +113,6 @@ Alternatively, community projects may concatenate names containing Mattermost, f
 
 ## Company Short Description
 
-Mattermost is an open source Slack-alternative built for enterprise. Thousands of companies use Mattermost for secure, compliant workplace messaging across web, PC and phones with archiving, search, corporate SSO and integration with over 700 third party applications. Available under MIT license in 16 languages, Mattermost offers peace-of-mind, value, control, and freedom from lock-in for organizations around the world.
-
-Learn more at [https://mattermost.com/](https://mattermost.com/)
+Mattermost is an open source Slack-alternative built for enterprise. Thousands of companies use Mattermost for secure, compliant workplace messaging across web, PC and phones with archiving, search, corporate SSO, and integration with over 700 third-party applications. Available under MIT license in 16 languages, Mattermost offers peace-of-mind, value, control, and freedom from lock-in for organizations around the world.
```

Comment:
I edited this for punctuation. If acceptable there may be other instances of this which we'd need to edit with the same changes so 4/5 on this as it reads better.

Reply:
@justinegeffen I found one other occurrence at, will submit a separate PR

Reply:
I've made additional edits on that PR which will need to be applied here. Will open a new PR for those.